config/prow/plugins.yaml

```diff
@@ -936,6 +938,15 @@ plugins:
     - project
     - require-matching-label
 
+  # Use the same settings as kubernetes/test-infra.
+  kubernetes-sigs/prow:
+    plugins:
+    - config-updater
+    - milestone
+    - override
+    - project
+    - require-matching-label
```

Comment:
This plugin won't do anything without configuration for the repo. We don't need such configuration at this time so please don't enable this plugin. https://github.com/kubernetes/test-infra/blob/21ad9e9c0b29d34d0520e6b94a2880eba9fe257e/config/prow/plugins.yaml#L550-L760